README.md

```diff
@@ -1,21 +1,54 @@
+# Node Modules List Webpack Plugin
+
 [![npm version](https://badge.fury.io/js/webpack-node-modules-list.svg)](https://badge.fury.io/js/webpack-node-modules-list)
 
-Exports meta data of all used node modules of a webpack bundle to a file.
+Exports metadata of all used node modules of a webpack bundle to a file.  
+This plugin works perfectly with the `SplitByPathPlugin` plugin.
```

Comment:
I would remove the reference to the split by path plugin here and add it to the example paragraph.